### PR TITLE
Tlotliso

### PR DESCRIPTION
**Pull request by TlotlisoMokatiJohn**

### CODE REVIEW

**Review by gr1d99 — CHANGES_REQUESTED** [review]
Screenshot 2021-08-30 at 11 54 50

getting this error

**Comment by TlotlisoMokatiJohn** [comment]
let me check if i made a typo sir ill get  back to you as soon as possible

File: src/components/Register.js [hunk 1]
```diff
@@ -1,46 +1,110 @@
...
+		}
+		);
+	};
 	return (
 		<div>
 			<div className="p-10 my-30 ">
-				<h2 className="text-center text-3xl text-blue-400">SIGN UP BELOW</h2>
-				<form action="">
+				<h2 className="sign-up">SIGN UP BELOW</h2>
+				<form action="" onClick={handleSubmit}>
```
Comment (gr1d99) [inline src/components/Register.js:59]: Use `onSubmit` event here not onClick

Comment (TlotlisoMokatiJohn) [reply src/components/Register.js]: I have changed it and I saw a big difference after updating it

**Review by gr1d99 — APPROVED** [review]
Screenshot 2021-08-31 at 11 27 30

There are no values stored on localstorage, have a look at it

**Comment by TlotlisoMokatiJohn** [comment]
Okay I realized that i did not set values from the hook since i was
debugging the form validation i'll get back to you when its fully done

**Review by gr1d99 — APPROVED** [review]

**Pull request merged**